iml-gui/crate/src/lib.rs

```diff
@@ -364,6 +364,10 @@ pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg, GMsg>)
         }
         Msg::LoadPage => {
             if model.loading.loaded() && !model.page.is_active(&model.route) {
+                if model.route == Route::Snapshots && !model.conf.use_snapshots {
+                    model.route = Route::NotFound;
```

Comment:
Maybe do it earlier on Msg::RouteChanged?

Reply:
It won't work because we need to wait for the conf to be loaded.